lib/service_actor/attributable.rb

```diff
@@ -27,6 +27,11 @@ def input(name, **arguments)
           result[name]
         end
 
+        # For avoid method redefined warning messages.
+        if method_defined?(name)
+          alias_method name, name
+        end
```

Comment:
Is there any advantage to do this inside `method_defined?` to calling it directly like so?

```suggestion
        alias_method name, name
```

Reply:
It's seem like only need run this alias_method if someone override on user code.

Reply:
👍🏻 

RuboCop suggests:

```suggestion
        alias_method name, name if method_defined?(name)
```

Reply:
Also, can you merge origin/main ?